### ISSUE TITLE
Any ideas on how to render footnotes as sidenotes like the tufte style?

### ISSUE THREAD

**Issue by trosel** [rust-lang/mdBook#1178]
I'm not sure how to go about this, but I want to render markdown footnotes as sidenotes like in tufte style https://edwardtufte.github.io/tufte-css/

Just looking for ideas

**Comment by rubenmoor** [comment]
Yes, I would use the same style for my footnotes. Conventional footnotes break the flow when reading. Side notes are a nice alternative.
